packages/react-next/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -475,6 +475,11 @@ export const CalloutContentBase = React.memo(
       return null;
     }
 
+    // We probably just switched focus context to another window. Don't rerender
```

Comment:
@khmakoto , @ecraig12345 , With the new dismiss on window blur logic, I'm seeing a bug in react-next callout where we dismiss properly, but callout pops up right after. My initial intuition was there's a refocus logic somewhere which triggers such a  behavior, although disabling `shouldRestoreFocus` on Popup didn't prevent it. Do you know what else could be causing CalloutContent.base to rerender after dismiss? Repro by disabling this logic block

Reply:
I'm not sure offhand. @MLoughry do you have any idea what might be going on since you just converted Callout?

Reply:
I'm unclear on the repro. I don't even see this window-blurring code in master....

Reply:
The changes are propose in this branch. To repro the rerender I describe above, you could just apply the changes in [current file](https://github.com/microsoft/fluentui/pull/14363/files/ef69a170f2e975d3e1bd736d433a151aa1323890..7353e4c19545f4d66e9d74034ed481f07c58914e#diff-d43dba3bcdb7a43545edf7be95f49465) on any branch you have locally, then comment out this block that return null on render(). With a callout open, clicking outside the page should trigger a dismiss, then you'll see the callout pop back up again